packages/kbn-monaco/scripts/utils/create_autocomplete_definitions.js

```diff
@@ -36,7 +36,7 @@ const getDisplayName = (name, imported) => {
     displayName = name.split('.').pop() || name;
   }
 
-  return displayName.replace('$', '.');
+  return displayName.replace(/^\$/g, '.');
```

Comment:
Switched this for a regex that only matches the beginning, because of the comment [here](https://github.com/elastic/kibana/pull/196678/files#diff-18e6f4f64a5666891a9b3acb0e735e1c84b80735b36831e67e58357b4f2c64fcR23-R24)